tests/jballerina-unit-test/src/test/resources/test-src/statements/arrays/array-test.bal

```diff
@@ -256,6 +256,22 @@ function testArrayTupleType() {
     test:assertEquals("typedesc [string[2],int,float[3][4]][][]", t.toString());
 }
 
+function testPrintableByteArrayInTable() {
```

Comment:
Shall we update the PR title and the name of the test to reflect what's actually being fixed/tested?

Reply:
Updated.